packages/atomic/src/components/atomic-pager/atomic-pager.tsx

```diff
@@ -29,52 +31,78 @@ export class AtomicPager implements InitializableComponent {
   @BindStateToController('pager')
   @State()
   private pagerState!: PagerState;
-  @State() public error!: Error;
+  @BindStateToI18n()
+  @State()
+  private strings = {
+    pagination: () => this.bindings.i18n.t('pagination'),
+    previous: () => this.bindings.i18n.t('previous'),
+    next: () => this.bindings.i18n.t('next'),
+    pageNumber: (page: number) => this.bindings.i18n.t('pageNumber', {page}),
+  };
+  @State() error!: Error;
+
+  /**
+   * Specifies how many page buttons to display in the pager.
+   */
+  @Prop() numberOfPages = 5;
+  /**
+   * Specifies whether the **Previous** and **Next** buttons should appear at each end of the pager when appropriate.
+   */
+  @Prop() enableNavigationButtons = true;
 
   public initialize() {
-    this.pager = buildPager(this.bindings.engine);
+    this.pager = buildPager(this.bindings.engine, {
+      options: {numberOfPages: this.numberOfPages},
+    });
   }
 
-  private get backButton() {
-    if (!this.pagerState.hasPreviousPage) {
-      return null;
-    }
-
-    const icon = '<';
+  private buildButton(options: {
```

Comment:
Buttons should probably include [`aria-current`](https://www.w3.org/TR/wai-aria-1.1/#aria-current), which seems to be recommended for pagination.